src/styles/card/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+  }
+
+  &-container {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+    align-items: center;
+  }
+
+  &-container__reviews {
```

Comment:
The same issue occurs with `stars-container__reviews`. According to BEM, this should be an element of `stars__container`, not `stars-container`. Consider renaming to `stars__container__reviews`.